### PR TITLE
New Resource `alicloud_ecs_invocation`; New Datasource `alicloud_ecs_invocations`;

### PR DESCRIPTION
**Pull request by super-eggs**
New Resource `alicloud_ecs_invocation`;
New Datasource `alicloud_ecs_invocations`;

### CODE REVIEW

File: alicloud/resource_alicloud_ecs_invocation_test.go [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+	command_content   = "ZWNobyBoZWxsbyx7e25hbWV9fQ=="
+	description       = "For Terraform Test"
+	type              = "RunShellScript"
+	working_dir       = "/root"
+	enable_parameter  = true
+}
+`, name)
+}
+
+func TestUnitAlicloudECSInvocation(t *testing.T) {
```
Comment (super-eggs) [inline alicloud/resource_alicloud_ecs_invocation_test.go:165]: 单测覆盖率 97.1%

File: alicloud/connectivity/regions.go [hunk 2]
```diff
@@ -192,3 +192,5 @@ var LogResourceSupportRegions = []Region{HeYuan}
 var AliKafkaSupportRegions = []Region{Beijing, CnNorth2Gov1, Qingdao, APSouthEast3, Huhehaote, APSouth1, EUWest1, ShenZhenFinance, ChengDu, USEast1, USWest1, Hangzhou, Zhangjiakou, Shenzhen, Shanghai, Hongkong, HeYuan, APSouthEast5, APNorthEast1, ShanghaiFinance, APSouthEast1, EUCentral1}
 var BastionhostSupportRegions = []Region{CnNorth2Gov1, Qingdao, ShanghaiFinance, EUCentral1, EUWest1, ChengDu, Shanghai, HeYuan, APNorthEast1, MEEast1, APSouth1, Hongkong, Zhangjiakou, USWest1, APSouthEast1, APSouthEast2, Huhehaote, APSouthEast5, Beijing, Hangzhou, ShenZhenFinance, APSouthEast3, USEast1, Shenzhen}
 var ACKSystemDiskEncryptionSupportRegions = []Region{Hongkong}
+
```
Comment (xinmu0426) [inline alicloud/connectivity/regions.go:195]: ECSSupportRegions 这个去掉吧

File: alicloud/data_source_alicloud_ecs_invocations.go [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+				Type:     schema.TypeList,
+				Optional: true,
+				ForceNew: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+				Computed: true,
+			},
+			"invoke_status": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
```
Comment (xinmu0426) [inline alicloud/data_source_alicloud_ecs_invocations.go:41]: 这个没有枚举值么

File: alicloud/resource_alicloud_ecs_invocation_test.go [hunk 4]
```diff
@@ -0,0 +1,356 @@
...
+		patches.Reset()
+		switch errorCode {
+		case "NonRetryableError":
+			assert.NotNil(t, err)
+		case "{}":
+			assert.Nil(t, err)
+		}
+	}
+
+	//// Update
```
Comment (xinmu0426) [inline alicloud/resource_alicloud_ecs_invocation_test.go:349]: 注释去掉

**Pull request merged**